dev/com.ibm.ws.ejbcontainer.mdb/src/com/ibm/ws/ejbcontainer/mdb/internal/MessageEndpointFactoryImpl.java

```diff
@@ -96,15 +101,14 @@ public MessageEndpointFactoryImpl() throws RemoteException {
     @Override
     public void initialize(EJSContainer ejsContainer,
                            BeanId id,
-                           BeanMetaData bmd)
-                    throws RemoteException
-    {
+                           BeanMetaData bmd) throws RemoteException {
         super.initialize(ejsContainer, id, bmd);
 
         // Get the activation spec name. This will have been obtained from the
         // binding file, or defaulted to "[app/]module/bean" conforming to the
         // java:global rules for binding an EJB.
         activationSvcId = bmd.ivActivationSpecJndiName;
+        name = "messageEndpoint-" + activationSvcId;
```

Comment:
I'm still not sure if this is what we want to use for the target name, but it is fine for now.